interactors/src/main/java/com/thefuntasty/interactors/disposables/CompletableDisposablesOwner.kt

```diff
@@ -65,8 +65,8 @@ interface CompletableDisposablesOwner {
             this@execute.currentDisposable?.dispose()
         }
 
-        val disposable = create(args)
-            .subscribe(
+        completablerConfig.onStart()
+        val disposable = create(args).subscribe(
```

Comment:
I'm fine with current solution, but have you thought about something similar to this:
```
val disposable = create(args)
    .doOnSubscribe { completablerConfig.onStart() }
    .subscribe(
        completablerConfig.onComplete,
        wrapWithGlobalOnErrorLogger(completablerConfig.onError)
    )
```

Reply:
Fixed 🙂